Makefile

```diff
@@ -90,7 +90,7 @@ lint:  ## cargo check and clippy. Skip clippy on guest code since it's not suppo
 	dprint check
 	cargo +nightly fmt --all --check
 	cargo check --all-targets --all-features
-	SKIP_GUEST_BUILD=1 cargo clippy --all-targets --all-features
+	SKIP_GUEST_BUILD=1 cargo clippy --all-targets --all-features -- --allow clippy::doc_overindented_list_items
```

Comment:
What's that?

Reply:
It's a little bit unexpected to see this in Makefile instead of in clippy.toml (so it's applied to both `make` and `cargo`) or in the code itself where we need it.